pkg/apis/elasticsearch/v1/elasticsearch_types.go

```diff
@@ -219,6 +219,7 @@ type NodeSet struct {
 	Name string `json:"name"`
 
 	// Config holds the Elasticsearch configuration.
+	// +kubebuilder:pruning:PreserveUnknownFields
```

Comment:
Which ones do you mean exactly?

Reply:
```diff
diff --git a/pkg/apis/elasticsearch/v1beta1/elasticsearch_types.go b/pkg/apis/elasticsearch/v1beta1/elasticsearch_types.go
index 3dd03c28..3230e1a2 100644
--- a/pkg/apis/elasticsearch/v1beta1/elasticsearch_types.go
+++ b/pkg/apis/elasticsearch/v1beta1/elasticsearch_types.go
@@ -61,6 +61,7 @@ type NodeSet struct {
        Name string `json:"name"`
 
        // Config holds the Elasticsearch configuration.
+       // +kubebuilder:pruning:PreserveUnknownFields
        Config *commonv1beta1.Config `json:"config,omitempty"`
 
        // Count of Elasticsearch nodes to deploy.
@@ -69,12 +70,14 @@ type NodeSet struct {
 
        // PodTemplate provides customisation options (labels, annotations, affinity rules, resource requests, and so on) for the Pods belonging to this NodeSet.
        // +kubebuilder:validation:Optional
+       // +kubebuilder:pruning:PreserveUnknownFields
        PodTemplate corev1.PodTemplateSpec `json:"podTemplate,omitempty"`
 
        // VolumeClaimTemplates is a list of persistent volume claims to be used by each Pod in this NodeSet.
        // Every claim in this list must have a matching volumeMount in one of the containers defined in the PodTemplate.
        // Items defined here take precedence over any default claims added by the operator with the same name.
        // +kubebuilder:validation:Optional
+       // +kubebuilder:pruning:PreserveUnknownFields
        VolumeClaimTemplates []corev1.PersistentVolumeClaim `json:"volumeClaimTemplates,omitempty"`
 }
```

Reply:
Ah I understand now. I don't think we do, because we don't use structural schema in v1beta1 and the default was `true` in v1beta1 for backwards compatibility afaik

Reply:
:+1: 

I admit I just:
```